src/components/NavPopover/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./NavPopover";
```

Comment:
I've been learning a lot from how the Chakra team structures their more complex components. Their strategy for promoting reusability with complex components while avoiding bloat to the API surface area is to split the component into smaller chunks - making it a compound component.

This follows that approach so that we can provide a mininal API for basic usecases, while also allowing developers to define their own NavPopoverTrigger or pass custom props to sub-components without creating a complex root component API